packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -154,7 +154,9 @@ export class ColumnHeaderCell extends AbstractComponent<IColumnHeaderCellProps,
 
         const nameComponent = (
             <LoadableContent loading={loading} variableLength={true}>
-                {renderName == null ? defaultName : renderName(name, index)}
+                {renderName == null
```

Comment:
Isn't multiline JSX forbidden by our TSLint config? Regardless, I'd prefer to kick this out into a variable above to keep the JSX lean.